src/pages/signin/LoginForm/LoginFormNarrow.js

```diff
@@ -99,7 +99,7 @@ class LoginFormNarrow extends React.Component {
                         {' '}
                         <Text
                             style={[styles.link, styles.mx1]}
-                            onPress={() => openURLInNewTab('https://github.com/Expensify/Expensify.cash')}
+                            onPress={() => openURLInNewTab('https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2')}
```

Comment:
I'd recommend we fix this lint warning by putting this URL into `CONST.js` and then accessing it. In `CONST.js` we might have to split the line into two lines to fix any additional lint warnings.